src/content/components/PriorityGuide/PriorityGuide.scss

```diff
@@ -56,6 +64,10 @@
   @media(max-width: 1210px) {
     display: none;
   }
+
+  &:hover {
+    cursor: pointer;
+  }
```

Comment:
Doesn't need to go in `:hover` - cursor will only apply when it's hovered anyway. So this actually just makes it a tiny bit slower, needlessly. I'd just put it in the rules in the block above